### PR TITLE
Add QuickSwap-deployed testnet contracts to DEX SDK

### PR DESCRIPTION
**Pull request by pano-skylakis**
# Summary

Updates the DEX SDK to use the QuickSwap-deployed Uniswap-V3 contracts on testnet, along with the most recent deployment of the ImmutableSwapProxy contract.

Verified using the DEX sample app.

Contract addresses can be found [here](https://immutable.atlassian.net/wiki/spaces/PROT/pages/2235205111/Quickswap)

### CODE REVIEW

**Review by Benjimmutable — APPROVED** [review]

File: packages/internal/dex/sdk-sample-app/src/config/devnet.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@ const immutableConfig = new ImmutableConfiguration({
 const contractOverrides: ExchangeContracts = {
   multicall: '0x9482D1727424B6C3EeaA22B037FFBC3ae6748f66',
   coreFactory: '0x8081d5F526b7Aaf4868e6C53Aa8a9d9D93c10562',
-  quoterV2: '0xC12B5c73951CFD922979638b5d19C593ac51dcDA',
-  peripheryRouter: '0x8089b5D6fa3f19C64081d5050c5CA3a66f34C5af',
+  quoter: '0xC12B5c73951CFD922979638b5d19C593ac51dcDA',
+  swapRouter: '0x8089b5D6fa3f19C64081d5050c5CA3a66f34C5af',
   secondaryFee: '0x8089b5D6fa3f19C64081d5050c5CA3a66f34C5af', // not deployed currently
```
Comment (keithbro-imx) [inline packages/internal/dex/sdk-sample-app/src/config/devnet.ts:20]: Should we rename this one to swapProxy? I actually thought i already did

Also why does it say "not deployed currently" ?

**Review by keithbro-imx — APPROVED** [review]
Looks good, no blockers, just the comment and the suggestion left in slack https://imtbl.slack.com/archives/C03DFBLQXDF/p1704753457406819?thread_ts=1704751256.706449&cid=C03DFBLQXDF

File: packages/internal/dex/sdk-sample-app/src/config/devnet.ts (hunk 1 above)
Comment (pano-skylakis) [reply packages/internal/dex/sdk-sample-app/src/config/devnet.ts]: Deployed here using ol faithful https://blockscout.dev.immutable.com/address/0x0234ceca85Efb0c3a751088d328F3db3d397DDBF/contracts#address-tabs

Comment (pano-skylakis) [reply packages/internal/dex/sdk-sample-app/src/config/devnet.ts]: Also this PR has your changes but wasn't merged, so doing that now https://github.com/immutable/ts-immutable-sdk/pull/1315

**Review by keithbro-imx — APPROVED** [review]

**Review by Benjimmutable — APPROVED** [review]

**Pull request merged**